test/Microsoft.Extensions.ProjectModel.Tests/Microsoft.Extensions.ProjectModel.Tests.csproj

```diff
@@ -23,17 +22,12 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Build.Runtime" Version="15.1.548" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0-preview-20161123-03" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
     <PackageReference Include="Microsoft.DotNet.Cli.Utils" Version="1.0.0-preview4-004233" />
     <PackageReference Include="Microsoft.DotNet.InternalAbstractions" Version="1.0.0" />
```

Comment:
This was a workaround for early versions of xunit and mostly likely can be removed.